test/performance/benchmarks/rollout-probe/main.go

```diff
@@ -229,5 +230,12 @@ func checkSLA(results *vegeta.Metrics) error {
 		return fmt.Errorf("SLA 1 failed. P95 latency is not in 100-110ms time range: %s", results.Latencies.P95)
 	}
 
+	// SLA 2: making sure the defined vegeta rates/total requests are met
+	if results.Rate == rate.Per.Seconds() {
```

Comment:
same comments as above

Reply:
I am a bit confused here. Should not this be: `if results.Rate == rate.Rate(time.Second) ` like in the case of the dataplane-probe?

Btw this is misleading:
```
func (cp ConstantPacer) Rate(elapsed time.Duration) float64 {
	return cp.hitsPerNs() * 1e9
}
```

Reply:
Yes, It should be rate.Rate(time.Second)